src/services/PaymentService.ts

```diff
@@ -184,7 +184,7 @@ export class PaymentService {
       currency: price.currency,
       interval: price.recurring!.interval as 'year' | 'month',
       nextPayment: subscription.current_period_end,
-      amountAfterCoupon: upcomingInvoice.total !== price.unit_amount ? upcomingInvoice.total : undefined,
+      amountAfterCoupon: upcomingInvoice.total,
```

Comment:
Maybe we can remove this parameter and add the upcomingInvoice.total in the "amount" parameter?

Reply:
Lets keep this parameter as maybe in the future, we have the possibility to show before/after price open.